mldsa/polyvec.h

```diff
@@ -562,6 +595,17 @@ __contract__(
 );
 
 #define mld_polyveck_pack_t0 MLD_NAMESPACE(polyveck_pack_t0)
+/*************************************************
+ * Name:        mld_polyveck_pack_t0
+ *
+ * Description: Bit-pack polynomial vector t0 to byte array of
+ *              MLDSA_K * MLDSA_POLYT0_PACKEDBYTES with coefficients
+ *              in [-2^{MLDSA_D-1},2^{MLDSA_D-1}].
```

Comment:
The bound here is actually wrong - the lower bound is exclusive.

```suggestion
 * Description: Bit-pack polynomial vector with coefficients in  ]-2^{MLDSA_D-1}, 2^{MLDSA_D-1}].
```

Reply:
got it,  wrong bound fixed.